library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.html

```diff
@@ -1595,3 +1597,176 @@ <h2 class="mat-subheading-2 flex-1 min-w-0 margin-remove-bottom" i18n>{{'cardcon
+                  <img mat-card-image [src]="widgetData.content.appIcon | pipePublicURL" loading="lazy"
+                  class="card-img ws-mat-primary-lite-background margin-remove"  [wsUtilsDefaultThumbnail]="defaultThumbnail"
+                  [alt]="widgetData.content.name" />
+                </ng-container>
+                <ng-template #defaultImg>
+                  <img mat-card-image [src]="defaultThumbnail" loading="lazy"
+                  class="card-img ws-mat-primary-lite-background margin-remove"
+                  [alt]="widgetData.content.name" />
+                </ng-template>
+              </ng-container>
+              <div class="source-div" style="display: none;">
+                <img [src]="widgetData.content.creatorLogo" class="source-icon" [wsUtilsDefaultThumbnail]="defaultSLogo"
+                  [alt]="(widgetData.content.sourceName + '_' + widgetData.content.identifier)" />
+              </div>
+              <span class="absolute resourse-type">
+                <img class="stats-icon " height="16" src="{{widgetData.content.mimeType === 'video/mp4'? '/assets/icons/home/play.svg': '/assets/icons/home/pdf.svg'}}" width="16">
+                {{widgetData.content.mimeType === "video/mp4"? 'Video': 'PDF'}}
+              </span>
+            </div>
+            <ng-container *ngIf="widgetData?.content?.cbPlanEndDate">
+              <div class="cbp-mark">
+                <span>{{widgetData?.content?.cbPlanEndDate | date: 'd MMM, y'}}</span>
+              </div>
+            </ng-container>
+            <!-- <ng-container *ngIf="widgetData?.content?.endDate">
+              <div class="cbp-mark">
+                <p *ngIf="widgetData?.content?.planDuration === acbpConstants?.UPCOMING"  class="cbp-warning">{{widgetData?.content?.endDate | date: 'd MMM, y'}}</p>
+                <p *ngIf="widgetData?.content?.planDuration === acbpConstants?.SUCCESS"  class="cbp-success">{{widgetData?.content?.endDate | date: 'd MMM, y'}}</p>
+                <p *ngIf="widgetData?.content?.planDuration === acbpConstants?.OVERDUE" class="cbp-danger">Over due</p>
+              </div>
+            </ng-container> -->
+            <ng-container *ngIf="cbPlanMapData && cbPlanMapData[widgetData?.content?.identifier]">
+              <ng-container [ngTemplateOutlet]="cbpPlan" [ngTemplateOutletContext]="{data: {
+                cbpData: cbPlanMapData[widgetData?.content?.identifier]
+              }}"></ng-container>
+            </ng-container>
+            <div class="duration-box right-corner" *ngIf="widgetData?.content?.duration > 0">
+              <mat-icon class="mat-icon main_icon mr-1">access_time</mat-icon>
+              <span class=" ws-mat-white-text duration">
+                {{ (widgetData?.content?.duration || 120)| pipeDurationTransform: 'hms' }}
+              </span>
+            </div>
+            <div class="duration-box right-corner" *ngIf="widgetData?.content?.programDuration > 0">
```

Comment:
we can check for "widgetData?.content?.programDuration" this one only instead > 0